modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/KotlinClientCodegen.java

```diff
@@ -54,8 +59,16 @@ public KotlinClientCodegen() {
         dateOptions.put(DateLibrary.THREETENBP.value, "Threetenbp");
         dateOptions.put(DateLibrary.STRING.value, "String");
         dateOptions.put(DateLibrary.JAVA8.value, "Java 8 native JSR310");
+        dateOptions.put(DateLibrary.JAVA8_LOCAL.value, "Java 8 using LocalDateTime");
         dateLibrary.setEnum(dateOptions);
         cliOptions.add(dateLibrary);
+        cliOptions.add(CliOption.newBoolean(USE_ASYNC, "Whether to use the Kotlin coroutines with the retrofit2 library."));
+
+        supportedLibraries.put(RETROFIT_2, "HTTP client: OkHttp 3.8.0. JSON processing: Gson 2.6.1 (Retrofit 2.3.0).");
```

Comment:
Version numbers shouldn't be encoded here, as they'll almost definitely become out of sync with templates. Also, users can provide custom templates with updated versioning.